src/Sylius/Bundle/ChannelBundle/composer.json

```diff
@@ -27,7 +27,9 @@
         "symfony/framework-bundle": "^2.7.7"
     },
     "require-dev": {
+        "matthiasnoback/symfony-dependency-injection-test": "^0.7.6",
```

Comment:
IIRC Sylius' main `composer.json` is sorted in that way, so I decided to stick to that.